FlowCrypt/src/main/res/values-uk/strings.xml

```diff
@@ -0,0 +1,487 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <string name="prompt_subject">Тема</string>
+    <string name="send">Відправити</string>
+    <string name="error_email_is_not_valid">Помилка: невірна Email адреса</string>
+    <string name="error_some_email_is_not_valid">Помилка: невірний Email \"%1$s\"</string>
+    <string name="internet_connection_is_not_available">Помилка: намає підключення до Інтернету</string>
+    <string name="unknown_error">Невідома помилка</string>
+    <string name="security">Безпека</string>
+    <string name="settings">Налаштування</string>
+    <string name="contacts">Контакти</string>
+    <string name="keys">Ключи</string>
+    <string name="help">Підтримка</string>
+    <string name="the_key_already_added">Такий ключ вже добавлено</string>
+    <string name="help_feedback_or_question">Допомога, питання, зворотній зв\'язок</string>
+    <string name="i_will_usually_reply_within_an_hour_except_when_i_sleep_tom">Дякуємо, що користуєтеся FlowCrypt! Ми тут, щоб допомогти Вам.</string>
```

Comment:
Here, hardcoded "FlowCrypt" probably should be changed to formatting placeholder and usage if this string should format it properly.